### PR TITLE
DOC-2300: Replace/rename Advanced Templates with Template in TinyMCE 7.0 docs.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2300

Site: [DOC-2300_DOC-2310 site](http://docs-feature-70-doc-2300doc-2310.staging.tiny.cloud/docs/tinymce/latest/advanced-templates/)

Changes:
* Replace or rename Advanced Templates with Template in TinyMCE 7.0 docs for DOC-2310
* update `:leveloffset: +1` to fix broken TOC for options.

Pre-checks:
- [x] Branch prefixed with `feature/7/` or `hotfix/7/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files removed have been deleted, not just excluded from the build `(if applicable)`

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/advanced-templates.adoc [hunk 1]
```diff
@@ -1,41 +1,44 @@
...
 
+[TIP]
+As of {productname} 7.0, the Advanced Template plugin has been renamed to {pluginname}. When adding {pluginname} in your editor, continue to use {plugincode}.
+
 The {pluginname} Premium plugin enables the creation of complex, interactive templates using {productname}.
 
 They can also be added to a {productname} instance by end-users selecting content in a {productname} document and saving the selection as a template via the {productname} user-interface.
 
-IMPORTANT: This initial {pluginname} release does not support template editing. To update a particular template; insert the template into an editor; make edits; select the, now edited, content; and then create a new template (using, for example, the *Tools > Save as Template…* menu item).
+// IMPORTANT: This initial {pluginname} release does not support template editing. To update a particular template; insert the template into an editor; make edits; select the, now edited, content; and then create a new template (using, for example, the *Tools > Save as Template…* menu item).
```
Comment (kemister85) [inline modules/ROOT/pages/advanced-templates.adoc:21]:
```suggestion
IMPORTANT: {pluginname} does not support template editing. To update a particular template; insert the template into an editor; make edits; select the, now edited, content; and then create a new template (using, for example, the *Tools > Save as Template…* menu item).
```

File: modules/ROOT/pages/advanced-templates.adoc [hunk 2]
```diff
@@ -1,41 +1,44 @@
-= Advanced Template plugin
-:navtitle: Advanced Template
+= Template plugin
```
[the remark this answers is not in the document]
Comment (kemister85) [reply modules/ROOT/pages/advanced-templates.adoc:1]:
```suggestion
= Templates plugin
```

File: modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -326,6 +325,7 @@
 **** xref:custom-dictionaries-for-tiny-spellchecker.adoc[Adding custom dictionaries]
 *** xref:autocorrect.adoc[Spelling Autocorrect]
 *** xref:tableofcontents.adoc[Table of Contents]
+*** xref:advanced-templates.adoc[Template]
```
Comment (kemister85) [inline modules/ROOT/nav.adoc:328]:
```suggestion
*** xref:advanced-templates.adoc[Templates]
```

File: modules/ROOT/partials/index-pages/premium-plugins.adoc [hunk 4]
```diff
@@ -147,6 +141,12 @@ xref:autocorrect.adoc[Spelling Autocorrect]
 
 Add autocorrection of common typos and spelling errors as well as capitalization correction to {productname}.
 
+a|
+[.lead]
+xref:advanced-templates.adoc[Template]
```
Comment (kemister85) [inline modules/ROOT/partials/index-pages/premium-plugins.adoc:146]:
```suggestion
xref:advanced-templates.adoc[Templates]
```

File: modules/ROOT/pages/mergetags.adoc [hunk 5]
```diff
@@ -101,17 +101,17 @@ Here is an example of the {pluginname} HTML structure.
 </span>
 ----
 
-== Merge Tags and the Advanced Templates Insertion Point Marker
+== Merge Tags and the Template Insertion Point Marker
 
 include::partial$misc/admon-requires-6.7v.adoc[]
 
-The xref:advanced-templates.adoc[Advanced Templates] Premium plugin can use a fixed string — `+{{mce-cursor}}+` — to xref:advanced-templates.adoc#the-insertion-point-marker[set the insertion point] within a template as the template is added to a {productname} document.
+The xref:advanced-templates.adoc[Template] Premium plugin can use a fixed string — `+{{mce-cursor}}+` — to xref:advanced-templates.adoc#the-insertion-point-marker[set the insertion point] within a template as the template is added to a {productname} document.
```
Comment (kemister85) [inline modules/ROOT/pages/mergetags.adoc:108]:
```suggestion
The xref:advanced-templates.adoc[Templates] Premium plugin can use a fixed string — `+{{mce-cursor}}+` — to xref:advanced-templates.adoc#the-insertion-point-marker[set the insertion point] within a template as the template is added to a {productname} document.
```

File: modules/ROOT/pages/mergetags.adoc [hunk 6]
```diff
@@ -101,17 +101,17 @@ Here is an example of the {pluginname} HTML structure.
...
 
 include::partial$misc/admon-requires-6.7v.adoc[]
 
-The xref:advanced-templates.adoc[Advanced Templates] Premium plugin can use a fixed string — `+{{mce-cursor}}+` — to xref:advanced-templates.adoc#the-insertion-point-marker[set the insertion point] within a template as the template is added to a {productname} document.
+The xref:advanced-templates.adoc[Templates] Premium plugin can use a fixed string — `+{{mce-cursor}}+` — to xref:advanced-templates.adoc#the-insertion-point-marker[set the insertion point] within a template as the template is added to a {productname} document.
 
 This fixed string uses the same default delimiting characters as individual merge tags use by default. It does not, however, interfere or otherwise interact with any {pluginname} configuration.
 
-The Advanced Templates plugin removes the Insertion Point Marker string before inserting a template containing the string in to a {productname} instance.
+The Template plugin removes the Insertion Point Marker string before inserting a template containing the string in to a {productname} instance.
```
Comment (kemister85) [inline modules/ROOT/pages/mergetags.adoc:112]:
```suggestion
The Templates plugin removes the Insertion Point Marker string before inserting a template containing the string in to a {productname} instance.
```

File: modules/ROOT/pages/available-toolbar-buttons.adoc [hunk 7]
```diff
@@ -230,6 +224,12 @@ include::partial$misc/plugin-toolbar-button-id-boilerplate.adoc[]
 :plugincode: table
 include::partial$misc/plugin-toolbar-button-id-boilerplate.adoc[]
 
+:plugincategory: premium
+:pluginname: Template
```
Comment (kemister85) [inline modules/ROOT/pages/available-toolbar-buttons.adoc:228]:
```suggestion
:pluginname: Templates
```

File: modules/ROOT/pages/advanced-templates.adoc [hunk 8]
```diff
@@ -122,7 +125,8 @@ For users who work with a fixed template list that does not require modification
 The following interactive predefined list example provides guidance for this approach.
 
 [[predefined-list-config-example]]
-==== Configuring the Advanced Template plugin to use a static list of predefined templates.
+==== Configuring the Template plugin to use a static list of predefined templates.
```
Comment (kemister85) [inline modules/ROOT/pages/advanced-templates.adoc:128]:
```suggestion
==== Configuring the Templates plugin to use a static list of predefined templates.
```

File: modules/ROOT/examples/live-demos/full-featured/index.html [hunk 9]
```diff
@@ -54,13 +54,13 @@ <h2>Character strings to demonstrate some of the Advanced Typography plugin’s
 	<li>30C is 86F</li>
 </ul>
 
-<h2>Note on the included Advanced Template demonstration</h2>
+<h2>Note on the included Template demonstration</h2>
```
Comment (ltrouton) [inline modules/ROOT/examples/live-demos/full-featured/index.html:57]:
```suggestion
<h2>Note on the included Templates demonstration</h2>
```

File: modules/ROOT/examples/live-demos/full-featured/index.html [hunk 10]
```diff
@@ -54,13 +54,13 @@ <h2>Character strings to demonstrate some of the Advanced Typography plugin’s
...
 
-<p>The included Advanced Template demonstration uses the <a class="mceNonEditable" href="{{site-url}}/tinymce/6/advanced-templates/#advtemplate_list"><code>advtemplate_list</code></a> configuration option to return a local promise containing a basic Template structure with self-contained examples.</p>
+<p>The included Template demonstration uses the <a class="mceNonEditable" href="{{site-url}}/tinymce/6/advanced-templates/#advtemplate_list"><code>advtemplate_list</code></a> configuration option to return a local promise containing a basic Template structure with self-contained examples.</p>
```
Comment (ltrouton) [inline modules/ROOT/examples/live-demos/full-featured/index.html:59]:
```suggestion
<p>The included Templates demonstration uses the <a class="mceNonEditable" href="{{site-url}}/tinymce/6/advanced-templates/#advtemplate_list"><code>advtemplate_list</code></a> configuration option to return a local promise containing a basic Template structure with self-contained examples.</p>
```

File: modules/ROOT/pages/advanced-templates.adoc [hunk 11]
```diff
@@ -1,41 +1,44 @@
-= Advanced Template plugin
-:navtitle: Advanced Template
+= Templates plugin
+:navtitle: Template
 :description: Create and edit complex templates in {productname} for use with {productname}.
 :description_short: Create templates.
-:keywords: plugin, advanced template, templates, advtemplate.
-:pluginname: Advanced Template
+:keywords: plugin, Template, templates, advtemplate.
```
Comment (ltrouton) [inline modules/ROOT/pages/advanced-templates.adoc:5]:
```suggestion
:keywords: plugin, template, templates, advtemplate.
```

File: modules/ROOT/pages/advanced-templates.adoc [hunk 12]
```diff
@@ -132,7 +136,8 @@ If the intention is for the template list to be end-user modifiable, the {plugin
 The following interactive remote storage configuration example provides guidance on configuring the {pluginname} plugin to support user-modifiable template lists.
 
 [[remote-config-example]]
-==== Configuring the Advanced Template plugin to interact with a remote backend service via REST API.
+==== Configuring the Template plugin to interact with a remote backend service via REST API.
```
Comment (ltrouton) [inline modules/ROOT/pages/advanced-templates.adoc:139]:
```suggestion
==== Configuring the Templates plugin to interact with a remote backend service via REST API.
```

File: modules/ROOT/pages/mergetags.adoc [hunk 13]
```diff
@@ -101,17 +101,17 @@ Here is an example of the {pluginname} HTML structure.
 </span>
 ----
 
-== Merge Tags and the Advanced Templates Insertion Point Marker
+== Merge Tags and the Template Insertion Point Marker
```
Comment (ltrouton) [inline modules/ROOT/pages/mergetags.adoc:104]:
```suggestion
== Merge Tags and the Templates Insertion Point Marker
```

File: modules/ROOT/partials/index-pages/premium-plugins.adoc [hunk 14]
```diff
@@ -147,6 +141,12 @@ xref:autocorrect.adoc[Spelling Autocorrect]
 
 Add autocorrection of common typos and spelling errors as well as capitalization correction to {productname}.
 
+a|
+[.lead]
+xref:advanced-templates.adoc[Templates]
+
+Create and edit complex templates in {productname} for use with {productname}.
```
Comment (ltrouton) [inline modules/ROOT/partials/index-pages/premium-plugins.adoc:148]: The original sentence seems odd to mention {productname} i.e. TinyMCE twice
```suggestion
Create and edit complex templates in {productname}.
```

File: modules/ROOT/pages/mergetags.adoc [hunk 15]
```diff
@@ -101,17 +101,17 @@ Here is an example of the {pluginname} HTML structure.
...
+The xref:advanced-templates.adoc[Templates] Premium plugin can use a fixed string — `+{{mce-cursor}}+` — to xref:advanced-templates.adoc#the-insertion-point-marker[set the insertion point] within a template as the template is added to a {productname} document.
 
 This fixed string uses the same default delimiting characters as individual merge tags use by default. It does not, however, interfere or otherwise interact with any {pluginname} configuration.
 
-The Advanced Templates plugin removes the Insertion Point Marker string before inserting a template containing the string in to a {productname} instance.
+The Templates plugin removes the Insertion Point Marker string before inserting a template containing the string in to a {productname} instance.
 
-Consequently, the Advanced Templates Insertion Point Marker string is never seen by the {pluginname} plugin.
+Consequently, the Template Insertion Point Marker string is never seen by the {pluginname} plugin.
```
Comment (ltrouton) [inline modules/ROOT/pages/mergetags.adoc:114]:
```suggestion
Consequently, the Templates Insertion Point Marker string is never seen by the {pluginname} plugin.
```

**Review by ltrouton — APPROVED** [review]

**Review by kemister85 — COMMENTED** [review]
copy edits

**Pull request merged**